x-pack/metricbeat/module/autoops_es/utils/env.go

```diff
@@ -26,6 +26,16 @@ func intDefault(a string, defaults int) int {
 	return int(b)
 }
 
+func boolDefault(a string, defaults bool) bool {
+	val, err := strconv.ParseBool(a)
```

Comment:
We should use a better parameter name:

```suggestion
func boolDefault(boolString string, defaults bool) bool {
	val, err := strconv.ParseBool(boolString)
```